backend/routes/replies.ts

```
@@ -0,0 +1,386 @@
+  parentType: "post" | "reply"; // Type of the direct parent
+  rootPostId: string; // ID of the original post tree root
+  quote: Quote;
+  createdAt: string; // ISO 8601 Timestamp String (consider changing to number for sorting)
+}
+
+/**
+ * @route   POST /replies/createReply
+ * @desc    Creates a new reply
+ * @access  Authenticated
+ */
+router.post<{}, CreateReplyResponse, { text: string; parentId: string; quote: Quote }>('/createReply', authenticateToken, async (req, res) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)